server/base/src/main/java/org/apache/accumulo/server/metrics/MetricsConfiguration.java

```diff
@@ -148,46 +148,35 @@ public Configuration getMetricsConfiguration() {
   }
 
   private void loadConfiguration() {
-    String accumuloConfDir = getEnvironmentConfiguration().getString("ACCUMULO_CONF_DIR");
-    if (null != accumuloConfDir) {
-      // Try to load the metrics properties file
-      File mFile = new File(accumuloConfDir, metricsFileName);
-      if (mFile.exists()) {
-        if (log.isDebugEnabled())
-          log.debug("Loading config file: " + mFile.getAbsolutePath());
-        try {
-          xConfig = new XMLConfiguration(mFile);
-          xConfig.append(getEnvironmentConfiguration());
-          xConfig.addConfigurationListener(new MetricsConfigListener());
-          xConfig.setReloadingStrategy(new FileChangedReloadingStrategy());
-
-          // Start a background Thread that checks a property from the XMLConfiguration
-          // every so often to force the FileChangedReloadingStrategy to fire.
-          if (null == watcher || !watcher.isAlive()) {
-            watcher = new MetricsConfigWatcher();
-            watcher.start();
-          }
-          notFound = false;
-          alreadyWarned = false;
-        } catch (ConfigurationException ce) {
-          log.error("Error reading accumulo-metrics.xml file.");
-          notFound = true;
-          return;
-        }
-      } else {
-        if (!alreadyWarned)
-          log.warn("Unable to find metrics file: " + mFile.getAbsolutePath());
-        alreadyWarned = true;
-        notFound = true;
-        return;
-      }
-    } else {
+    URL metricsUrl = MetricsConfiguration.class.getClassLoader().getResource("accumulo-metrics.xml");
+    if (metricsUrl == null) {
       if (!alreadyWarned)
-        log.warn("ACCUMULO_CONF_DIR variable not found in environment. Metrics collection will be disabled.");
+        log.warn("accumulo-metrics.xml was not found on classpath. Metrics collection will be disabled.");
       alreadyWarned = true;
       notFound = true;
       return;
     }
+
+    try {
+      xConfig = new XMLConfiguration(metricsUrl);
```

Comment:
XMLConfiguration is historical. I am only changing how metrics configuration file is found.